resource-management/pkg/service-api/endpoints/constants.go

```diff
@@ -22,9 +22,12 @@ const (
 
 	//RegionlessResourcePath is the default api service url
 	RegionlessResourcePath = "/resource"
-	ListWatchResourcePath  = RegionlessResourcePath + "/{clientid}"
-	UpdateResourcePath     = RegionlessResourcePath + "/{clientid}" + "/addResource"
-	ReduceResourcePath     = RegionlessResourcePath + "/{clientid}" + "/reduceResource"
+
+	NodeStatusPath = "/nodes?"
```

Comment:
our initial design is using RegionlessResourcePath as root path, suggest to change to:
NodeStatusPath = RegionlessResourcePath +"/nodes?"

Reply:
There are already two types of paths:
1. "/clients", used for client registration
2. "/resource", used for resources get/update

Querying for node status is a different type of request, it does not belong to any of the above.
We might need to add additional layer to nodes for large amount of node summary. The external APIs are designed to be provided from another service. This one is an internal API used for quick performance test.

Reply:
Suggest to add TODO to finalize the url later

Reply:
> Suggest to add TODO to finalize the url later

Done.